libweston/backend-rdp/rdprail.c

```diff
@@ -4123,7 +4225,7 @@ rdp_rail_notify_app_list(void *rdp_backend, struct weston_rdprail_app_list_data
 			updateAppList.appIcon = &iconData;
 			updateAppList.flags |= RDPAPPLIST_FIELD_ICON;
 		}
-		peerCtx->applist_server_context->UpdateApplicationList(peerCtx->applist_server_context, &updateAppList);
+		peerCtx->applist_server_context->context->updateApplicationList(peerCtx->applist_server_context, &updateAppList);

```

Comment:
I'm hitting compile error on this line, so do you have update on https://github.com/microsoft/wslg/blob/main/rdpapplist/rdpapplist_server.h, too?

Reply:
Ouch, that was a search and replace error in a section of code that doesn't compile for me. The change is just wrong, and I've dropped it now.